exchanges/bitfinex/bitfinex.go

```diff
@@ -491,6 +494,72 @@ func (b *Bitfinex) GetSiteListConfigData(ctx context.Context, set string) ([]str
 	return resp[0], nil
 }
 
+// GetSiteInfoConfigData returns site configuration data by pub:info:{AssetType} as a map
+// path should be bitfinexInfoPairs or bitfinexInfoPairsFuture???
+// NOTE: See https://docs.bitfinex.com/reference/rest-public-conf
+func (b *Bitfinex) GetSiteInfoConfigData(ctx context.Context, assetType asset.Item) (map[currency.Pair]order.MinMaxLevel, error) {
+	var path string
+	switch assetType {
+	case asset.Spot:
+		path = bitfinexInfoPairs
+	case asset.Futures:
+		path = bitfinexInfoFuturePairs
+	default:
+		return nil, fmt.Errorf("invalid asset type for GetSiteInfoConfigData: %s", assetType)
+	}
+	url := bitfinexAPIVersion2 + path
+	var resp [][][]any
+
+	err := b.SendHTTPRequest(ctx, exchange.RestSpot, url, &resp, status)
+	if err != nil {
+		return nil, err
+	}
+	if len(resp) != 1 {
+		return nil, errors.New("response did not contain only one item")
+	}
+	pairs := map[currency.Pair]order.MinMaxLevel{}
```

Comment:
Is there any particular reason for this being a map? 
In my testing I'm seeing no duplicate fields returned, and you are not performing any map lookups, just appending. `MinMaxLevel` contains asset and pair information. I don't see why not return `[]order.MinMaxLevel`
Just to test, I added:
```go
if _, ok := pairs[pair]; ok {
			panic(pair.String())
		}
```
and no duplicate existed

In your wrapper function `UpdateOrderExecutionLimits`, you iterate over the response too. Iterating over a map is much slower than a slice

Reply:
completed